src/en/space-station-14/round-flow/proposals/station-ai.md

```diff
@@ -0,0 +1,162 @@
+
+While using the AI map vision, the crew can be seen via suit sensors, only if they have tracking on. Otherwise AI is unable to tell if someone is in the area without using cameras. A potential ability idea is to give cameras a motion detection cone that gets rendered for AI, and highlight entities within it - differently from tracked crew that enters it.
+
+The map vision should not use “#00FF00 on #000000” color scheme for purpose of reducing eye fatigue. Consider dark dark gray background as a default for areas that are not space exposed. Preferably implement a way to set custom colors for your UI.
+
+### Cameras
+
+Cameras cover almost all of the crew areas, places where most people work and exist within, to monitor their wellbeing and the state of the station. They should not be nearly free to build, and should require an autolathe producible board, some steel, LV wire.
+
+They should not be more durable than a person - a hard enough knock should deactivate it for a small period of time, while sustaining enough damage will keep it offline until repaired. A non-violent option for keeping cameras offline should be available, using cloth and/or gauze makes it impossible to see through a camera, taking a relatively long time to cover it completely (3 or 5 second doafter, balanced by it being a “stealth” option). A camera disabled in such a way should still be shown as active for AI and warden. Though it should have no motion detection cone and convey such explicitly.
+
+Power is required for cameras to work at full capacity, reducing their vision to half or third of their normal range when it is not available. Though power being out is a bigger issue for AI itself than just the cameras.
+
+## Borgs - Sync and control
+
+The AI has no way to affect anything physical (at least, a shackled one - not malf). Its primary way to do anything in that regard is borgs, and directing the crew. While the crew has own concerns, like air, health and such, borgs are hardier and are almost always available for AI to use.
+
+Borgs can opt into synchronizing their laws with AI, gaining map or camera view from AI (todo: detail the incentives). This does not make them less susceptible to ion storm law corruption, but the AI can fix the laws by rewriting them to its own at some cost.
+
+Emagging should forcefully but silently unlink a borg from the AI if it’s synced. If it’s not, AI is not relevant.
```

Comment:
This needs to be delved into a lot because there's edge cases for this. This also kinda conflicts with how we want emagged borgs to be found out: through roleplay,

If a borg is emagged, it should break the sync but still fake the sync for AI and crew, basically continuing a one-way sync. The borg gets its benefits it would normally get from being synced (remote control and camera vision). But do we want synced borgs to also have a camera the AI can view? Because if we want the camera vision to cut upon being emagged, it would be a bit obvious if a borg is emagged.

Alternatively we could keep borg vision for emagged borgs and have them attempt to fool the AI as much as possible while still following directives.

A person who attempts to EMAG a borg that is synced/hacked by a Malf AI should just fake the EMAG working, otherwise any poor thief/traitor could just "malfcheck" AI. It would also be interesting to not fake the interaction and instead have the EMAG fry, that way roleplay could occur: the AI and thief/syndie would have to barter with the AI to achieve its goals, otherwise they would rat. But that's in a perfect world of HRP.

Also PS: Make camera vision through borgs take bits/s. Have research for "runtime optimizations" which reduces the bits/s cost for actions. Could also make alternative research for Malf AI with further improvements so Malf AI has more effective power from the same amount of servers. But you could have Malf AI do something "risky" as a way to earn it.